### PR TITLE
add NerdyBot into spider

### PR DESCRIPTION
**Pull request by jnozsc**
http://nerdybot.com/

### CODE REVIEW

File: regexes.yaml [hunk 1]
```diff
@@ -4726,7 +4726,7 @@ device_parsers:
...
+  - regex: '(bot|zao|borg|DBot|oegp|silk|Xenu|zeal|^NING|CCBot|crawl|htdig|lycos|slurp|teoma|voila|yahoo|Sogou|CiBra|Nutch|^Java/|^JNLP/|Daumoa|Genieo|ichiro|larbin|pompos|Scrapy|snappy|speedy|spider|msnbot|msrbot|vortex|^vortex|crawler|favicon|indexer|Riddler|scooter|scraper|scrubby|WhatWeb|WinHTTP|bingbot|openbot|gigabot|furlbot|polybot|seekbot|^voyager|archiver|Icarus6j|mogimogi|Netvibes|blitzbot|altavista|charlotte|findlinks|Retreiver|TLSProber|WordPress|SeznamBot|ProoXiBot|wsr\-agent|Squrl Java|EtaoSpider|PaperLiBot|SputnikBot|A6\-Indexer|netresearch|searchsight|baiduspider|YisouSpider|ICC\-Crawler|http%20client|Python-urllib|dataparksearch|converacrawler|Screaming Frog|AppEngine-Google|YahooCacheSystem|fast\-webcrawler|Sogou Pic Spider|semanticdiscovery|Innovazion Crawler|facebookexternalhit|Google.*/\+/web/snippet|Google-HTTP-Java-Client|BlogBridge|IlTrovatore-Setaccio|InternetArchive|GomezAgent|WebThumbnail|heritrix|NewsGator|PagePeeker|Reaper|ZooShot|holmes|NerdyBot)'
```
Comment (russellwhitaker) [inline regexes.yaml]: You need to add test coverage to validate "NerdyBot" parses to "Spider" then.

Comment (jnozsc) [inline regexes.yaml]: thanks. just added it :)

Comment (russellwhitaker) [inline regexes.yaml]: Very cool, thanks.

**Comment by jnozsc** [comment]
do I need to add any other stuff to this pull request?

**Comment by jnozsc** [comment]
thoughts? cc @tobie @elsigh

**Comment by jnozsc** [comment]
I noticed I used a older version ua-parser before I create this pull request, I notice this patch is not necessary for the latest version of ua-parser.

Close it.

Thanks

**Pull request closed without merging**